en/formulaic_privacy_notice.md

```diff
@@ -15,7 +15,7 @@ We collect the following information directly from you when you use the Service:
 
 **Account information.** We may collect personal information, such as your name, username, contact information, and any other information that you provide to us in connection with the creation or administration of your account.
 
-**Formulas, Inputs and Outputs.** There are currently no LLM or similar AI-based platforms operated by or affiliated with Mozilla. However, the Service allows you to query selected LLMs offered by third parties (“Third-Party Models”) by inputting information into templates (“Formulas”) made available by the Service, as well as any Formulas that you yourself may choose to create and/or contribute to the Service. We process the content of your queries, as well as any amendments or updates you make to them, and any other content (including any media) you choose to upload to the Service (“Inputs”) in order to run them through Third-Party Models and generate results (“Outputs”). While we do not store any of this information, you can choose to export your Outputs to your computer. If you abandon a query or navigate away from our site, all Inputs and Outputs will be discarded by your browser and irretrievably lost.
+**Formulas, Inputs and Outputs.** There are currently no LLM or similar AI-based platforms operated by or affiliated with Mozilla. However, the Service allows you to query selected LLMs offered by third parties (“Third-Party Models”) by inputting information into templates (“Formulas”) made available by the Service, as well as any Formulas that you yourself may choose to create and/or contribute to the Service. We process the content of your queries, as well as any amendments or updates you make to them, and any other content (including any media) you choose to upload to the Service (“Inputs”) in order to run them through Third-Party Models and generate results (“Outputs”). We also store Inputs and Outputs in your acount, and you can choose to export your Outputs to your computer. 
```

Comment:
There is a type here (acount). Fixed in https://github.com/mozilla/legal-docs/commit/3519090ebff89cbc173a79f2a1afbaba1dfbbaec